src/main/java/de/openknowledge/fruehauf/domain/TodoRepository.java

```diff
@@ -0,0 +1,27 @@
+package de.openknowledge.fruehauf.domain;
+
+
+import de.openknowledge.fruehauf.infrastructure.stereotypes.Repository;
+
+import javax.persistence.EntityManager;
+import javax.persistence.PersistenceContext;
+import javax.persistence.TypedQuery;
+import javax.persistence.criteria.CriteriaBuilder;
+import javax.persistence.criteria.CriteriaQuery;
+import javax.persistence.criteria.Root;
+import java.util.List;
+
+@Repository
+public class TodoRepository {
+    @PersistenceContext(unitName = "default")
+    private EntityManager em;
+
+    public List<Todo> allTodos() {
```

Comment:
Bitte in `getAllTodos` umbenennen.

Reply:
Ok das ist der beste/richtige Name. Danke